src/app/layout.tsx

```diff
@@ -1,13 +1,21 @@
+'use client';
+import { ReactNode, useState } from 'react';
+
 import localFont from 'next/font/local';
 
-import type { Metadata } from 'next';
+import { QueryClient, QueryClientProvider } from '@tanstack/react-query';
+// import type { Metadata } from 'next';
 
 import Header from '@/components/header';
+
+// Styles
+import 'maplibre-gl/dist/maplibre-gl.css';
 import '@/styles/globals.css';
-export const metadata: Metadata = {
-  title: 'Hub - Open-Earth-Monitor Cyberinfrastructure',
-  description: '...',
-};
+
+// export const metadata: Metadata = {
+//   title: 'Hub - Open-Earth-Monitor Cyberinfrastructure',
+//   description: '...',
+// };
```

Comment:
Try to not keep comments in the final version of the PR

Reply:
I am not sure if it's the final